osu.Game/Rulesets/Difficulty/Skills/Skill.cs

```diff
@@ -41,7 +40,11 @@ public abstract class Skill
         protected readonly LimitedCapacityStack<DifficultyHitObject> Previous = new LimitedCapacityStack<DifficultyHitObject>(2); // Contained objects not used yet
 
         private double currentStrain = 1; // We keep track of the strain level at all times throughout the beatmap.
+        private double dummyStrain = 1; // Balancing touchscreen strain so that effects are smaller.
+        private double prevDummyStrain = 1; // Balancing touchscreen strain so that effects are smaller.
+        private double previous = 0;
         private double currentSectionPeak = 1; // We also keep track of the peak strain level in the current section.
+        private double dummySectionPeak = 1; // Balancing touchscreen strain so that effects are smaller.
```

Comment:
These comments aren't really helpful, see my other comment.

Reply:
Issue has been resolved as a result of reverting strain calculation change completely.